### PR TITLE
EDX-2781 Add new input for go-lint workflow to install required packages

### PR DESCRIPTION
**Pull request by jinlinGuan**
Signed-off-by: Ginny Guan <ginny@iotechsys.com>

### CODE REVIEW

**Review by cloudxxx8 — APPROVED** [review]
LGTM

File: .github/workflows/reusable-golangci-lint.yml [hunk 1]
```diff
@@ -34,7 +38,11 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           # Retrieve go version from go.mod
-          go-version-file: "${{ inputs.WORKING_DIRECTORY }}/go.mod"     
+          go-version-file: "${{ inputs.WORKING_DIRECTORY }}/go.mod"   
+      - name: Install Required Packages
+        if: ${{ inputs.REQUIRED_PACKAGES != '' }}
+        run: |
+          sudo apt update && sudo apt-get install ${{ inputs.REQUIRED_PACKAGES }}
```
Comment (cloudxxx8) [inline .github/workflows/reusable-golangci-lint.yml:45]: use `apt` instead of `apt-get`

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: .github/workflows/reusable-golangci-lint.yml (hunk 1 above)
Comment (jinlinGuan) [reply .github/workflows/reusable-golangci-lint.yml]: updated

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**